pkgs/by-name/re/recoverdm/package.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, installShellFiles
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "recoverdm";
+  version = "0.20";
+
+  src = fetchFromGitHub {
+    owner = "D3vil0p3r";
+    repo = "recoverdm";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-te/MCjY/O4Mq7n3ZxxvOb20S9usgPm5DVc6oqNXkizU=";
+  };
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  makeFlags = [
+    "-C src"
+  ] ++ lib.optional stdenv.isDarwin "LDFLAGS=-lcrypto";
```

with patch darwin compiles bug generates a linker error as it can't find `-lcrypto`. i removed it and build succeeded so don't think this is necessary.
```suggestion
  ] ++ lib.optional stdenv.isDarwin "LDFLAGS=-lcrypto";
```

---

I used `lcrypto` because in the old webpage of the project there was reported something like `For MacOS X users, add -lcrypto to LDFLAGS in Makefile.`. Currently I removed it as requested.